File: ympa_fsharp_giraffe_server/src/model/GetOutletsResponse.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.FlippingPagerDTO
open ympa_fsharp_giraffe_server.Model.FullOutletDTO
open ympa_fsharp_giraffe_server.Model.ScrollingPagerDTO

module GetOutletsResponse =

  //#region GetOutletsResponse


  type GetOutletsResponse = {
    Outlets : FullOutletDTO[];
    Paging : ScrollingPagerDTO;
    Pager : FlippingPagerDTO;
  }
  //#endregion

File: src/main/scala/org/openapitools/server/model/GetOutletsResponse.scala
package org.openapitools.server.model


/**
 * Ответ на запрос информации о точках продаж.
 *
 * @param outlets Информация о точках продаж. for example: ''null''
 * @param paging  for example: ''null''
 * @param pager  for example: ''null''
*/
final case class GetOutletsResponse (
  outlets: Seq[FullOutletDTO],
  paging: Option[ScrollingPagerDTO] = None,
  pager: Option[FlippingPagerDTO] = None
)

